hack/kind/agent-svc-config.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: agent-install.openshift.io/v1beta1
```

Comment:
assisted-service k8s manifests are usually in `/deploy`, I think it would be better to add it there